test/jdk/javax/swing/JEditorPane/4330998/JEditorPaneSetTextNullTest.java

```diff
@@ -0,0 +1,55 @@
+ /*
+  * Copyright (c) 2013, 2022, Oracle and/or its affiliates. All rights reserved.
+  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+  *
+  * This code is free software; you can redistribute it and/or modify it
+  * under the terms of the GNU General Public License version 2 only, as
+  * published by the Free Software Foundation.
+  *
+  * This code is distributed in the hope that it will be useful, but WITHOUT
+  * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+  * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+  * version 2 for more details (a copy is included in the LICENSE file that
+  * accompanied this code).
+  *
+  * You should have received a copy of the GNU General Public License version
+  * 2 along with this work; if not, write to the Free Software Foundation,
+  * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+  *
+  * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+  * or visit www.oracle.com if you need additional information or have any
+  * questions.
+  */
+
+ import javax.swing.JEditorPane;
+ import javax.swing.JPanel;
+ import javax.swing.SwingUtilities;
+
+ /*
+  * @test
+  * @bug 4330998
+  * @summary Verifies that JEditorPane.setText(null) doesn't throw NullPointerException.
+  * @run main JEditorPaneSetTextNullTest
+  */
+ public class JEditorPaneSetTextNullTest {
+     private static JEditorPane editorPane;
+
+     public static void main(String[] s) throws Exception {
+         SwingUtilities.invokeAndWait(() -> createUI());
+
+         try {
+             SwingUtilities.invokeAndWait(() -> editorPane.setText(null));
```

Comment:
I believe the test could be simplified to
```suggestion
             SwingUtilities.invokeAndWait(() -> new JEditorPane().setText(null));
```
And `createUI()` can be removed.

Reply:
If you like to make sure the test expects `NullPointerException` only, you can use try-catch inside the lambda expression and propagate the exception thrown from `invokeAndWait`. Yet it's not required: we don't expect any exceptions thrown. Therefore try-catch can be dropped too.

Reply:
> But if we want to throw the RuntimeException with a failure message, I think we need to catch it, right?

Yes, if you want to provide a more specific error message.